day1/index.html

```diff
@@ -0,0 +1,46 @@
+<!doctype html>
+<html lang="en">
+
+  <head>
+    <title>Day 1</title>
+
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="index.css">
+  </head>
+
+  <body>
+    <div class="block">
+
+    <h1>1.) What is color is <font color="red">THIS</font>?</h1>
+
+    <br>
+    <button class="red" onclick="alert('Correct!')">Red</button>
+    <button class="blue" onclick="alert('Try again!')">Blue</button>
+    <br>
+
+    <h1><span class="ans" onclick="alert('Correct!')">2</a>.) 1 + 1 = ?</h1>
+    <br>
```

Comment:
Would be better to put all listeners in index.js

Reply:
I move all listeners to index.js and delete in index.html